torch/optim/adagrad.py

```diff
@@ -44,28 +45,18 @@ def __init__(
             foreach=foreach,
             maximize=maximize,
             differentiable=differentiable,
+            capturable=capturable,
         )
         super().__init__(params, defaults)
 
-        for group in self.param_groups:
```

Comment:
Thanks to this change, a previous error in test/nn/test_lazy_modules.py for lazy initialization is now avoided.